docs/includes/sso-yeoman-project-structure.md

```diff
@@ -14,18 +14,20 @@ The following files define the add-in's task pane UI and functionality.
 
 - The **./src/taskpane/taskpane.css** file contains the CSS that's applied to content in the task pane.
 
-- In a JavaScript project, the **./src/taskpane/taskpane.js** file contains code to initialize the add-in. In a TypeScript project, the **./src/taskpane/taskpane.ts** file contains code to initialize the add-in and also code that uses the Office JavaScript API library to add the data from Microsoft Graph to the Office document.
+- The **./src/taskpane/taskpane.js** file contains code to initialize the add-in and also code that uses the Office JavaScript API library to add the data from Microsoft Graph to the Office document.
```

Comment:
is there no longer a .ts file?

Reply:
Yes there is, but the pattern is to call out the *.ts files only when they differ from the corresponding *.js file.